pkg/dhclient/dhclient.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013-2019 the Webboot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package dhclient
+
+import (
+	"context"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/u-root/u-root/pkg/dhclient"
+	"github.com/vishvananda/netlink"
+)
+
```

Comment:
you need a comment on this New function since it is exported. 
Be sure to run golint and go vet on these files before you push again.

Reply:
Also, New by convention returns data structures or interfaces. This is not a New. This is actually doing the dhcp setup. It should have a different name.

Reply:
+1 on this. When I call something called dhclient.New, I expect that function to create a new instance of dhclient, not to do the whole protocol. You could call it something like Request or something.

Reply:
that is very true. I will try to come up with a better name for this function.